src/app/commons/IndexBanner/IndexBanner.js

```diff
@@ -0,0 +1,26 @@
+import './IndexBanner.scss'
+import {NavLink} from "react-router-dom";
+import FakeLink from "../FakeLink";
+
+function link(currentPathName, to, name, Icon) {
+    return (
+        <NavLink to={to} activeClassName="is-active">
+            <Icon className="tab-icon"/>{name}</NavLink>
+    );
+}
+
+const IndexBanner = function ({currentPathName, path, tabContents}) {
```

Comment:
1. This is not a Banner
this is a navigation bar (or any better ideas?).

Rename it to `InPageNavigationBar` (remember to rename all its related files e.g., css)

2. A comment that documents how to use this component is required